apps/src/blockly/eventHandlers.js

```diff
@@ -92,3 +93,19 @@ export function adjustCalloutsOnViewportChange(event) {
     handleWorkspaceResizeOrScroll();
   }
 }
+
+// When the browser is resized, we need to re-adjust the width of any open flyout.
+export function reflowToolbox(event) {
+  const mainWorkspace = Blockly.getMainWorkspace();
+  const mainWorkspaceFlyout = findFlyout(mainWorkspace);
+  if (mainWorkspaceFlyout) {
+    mainWorkspaceFlyout.reflow();
+  }
+  if (Blockly.functionEditor) {
+    const modalWorkspace = Blockly.functionEditor.editorWorkspace;
```

Comment:
you can use `Blockly.getFunctionEditorWorkspace()` instead ([link](https://github.com/code-dot-org/code-dot-org/blob/5edb06bff6a25949911a493f67a6ec07b0c06178/apps/src/blockly/googleBlocklyWrapper.js#L764)).